internal/core/software_renderer.rs

```diff
@@ -1181,6 +1181,11 @@ impl<B: target_pixel_buffer::TargetPixelBuffer> RenderToBuffer<'_, B> {
 
     fn process_texture_impl(&mut self, geometry: PhysicalRect, texture: SceneTexture<'_>) {
         self.foreach_region(&geometry, |buffer, rect, extra_left_clip, extra_right_clip| {
+            let tex_src_off_x = (texture.extra.off_x + Fixed::from_integer(extra_left_clip as u16))
+                * texture.extra.dx.truncate();
```

Comment:
I'll make a follow-up to try to avoid truncate() here but combine the two different fixed formats.